pkgs/applications/misc/kaufkauflist/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, buildNpmPackage, fetchFromGitea, nix-update-script }:
+
+buildNpmPackage rec {
+  pname = "kaufkauflist";
+  version = "1.0.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "annaaurora";
+    repo = "kaufkauflist";
+    rev = "v${version}";
+    sha256 = "sha256-c5BKBm8/Mif7XGLMHTvFTXzOXsU2loDqcTgEZa6t4rY=";
+  };
+
+  npmDepsHash = "sha256-cEkInUAniBwOH4//MisffXest9KAmT0M5dnAdGcQ9+o=";
+
+  buildPhase = ''
+    runHook preBuild
+
+    npm run build
+
+    runHook postBuild
+  '';
```

Comment:
Isn't this the default?

Reply:
oh, yeah it is. Just tested it and will force push without that.